### ISSUE TITLE
About save png image

### ISSUE THREAD

**Issue by MontaEllis** [fepegar/torchio#411]
My training set and testing set are both png image. While inferencing, the output of :
`output_image = torchio.ScalarImage(tensor=(output_tensor_1*255.).numpy(), affine=patches_batch['mri']['affine'][0].numpy())
 output_image.save(os.path.join(output_dir_test_float,"label"+"_"+str(i+1)+".png"))`
is 180 flop from the ground truth. Why it happens?

**Comment by fepegar** [comment]
Hi, @MontaEllis. Can you please follow the guidelines in the "bug" issue template? 
That will help me understand the problem so I can help you.

https://github.com/fepegar/torchio/issues/new?assignees=&labels=bug&template=bug_report.md&title=

**Comment by fepegar** [comment]
For now, here's a refactored version of the code:

```python
tensor = (output_tensor * 255.).numpy()
affine = patches_batch['mri']['affine'][0].numpy()
output_image = torchio.ScalarImage(tensor=tensor, affine=affine)
output_image.save(path)
```

I am not sure what do you mean by "180 flop from the ground truth".

**Comment by MontaEllis** [comment]
Here is the source image: ![](https://ellis.oss-cn-beijing.aliyuncs.com/img/20210103093701.png)
Here is the label image : ![](https://ellis.oss-cn-beijing.aliyuncs.com/img/20210103093822.png)
After inference,when I save the float result as png : ![](https://ellis.oss-cn-beijing.aliyuncs.com/img/20210103093905.png)
when I save the float result as mhd: ![](https://ellis.oss-cn-beijing.aliyuncs.com/img/20210103093958.png)
You can see, the mhd and source has the same location, while the png is 180 flop from the source

**Comment by fepegar** [comment]
It's hard to say without seeing more code. It would help –a lot– if you could provide a minimally working example to reproduce this. I don't know if you are using `RandomFlip`, or what's in `patches_batch['mri']['affine']`. Can you share the images and some code?

**Comment by fepegar** [comment]
Closing this issue for now. Feel free to reopen if you can provide more information.
